flocker/node/functional/test_docker.py

```diff
@@ -138,6 +140,46 @@ def start_container(self, unit_name,
 
         return d
 
+    def test_force_remove(self):
+        """
+        ``DockerClient.remove`` can forcefully remove a running container
+        when the ``force`` parameter is set to True.
+        """
+        client = DockerClient()
+        unit_name = u'test-force-remove'
+        expected_states = (u'active',)
+        d = client.add(
+            unit_name=unit_name,
+            image_name=u'clusterhq/flask'
```

Comment:
This image and flask are incidental complexity for this test.  The detail of which container to run doesn't really matter to what the test is concerned with testing.  All of the code from Flask that will run when this test runs is likewise irrelevant.  Depending on the execution environment of the test suite, it's also possible this will result in a Docker image being pulled from Dockerhub.

The incidental complexity increases the maintenance cost of the test method.  The Dockerhub dependency increases the chances that this will fail intermittently due to transient network or Dockerhub conditions.

A way to address the incidental complexity is to contain it and then isolate it from this particular test.  Add an attribute or method to the class responsible for giving back the name of an image that has the desired properties - in this case, an image that runs indefinitely, I think.

A way to address the external dependency is to build a custom image for this test as we do elsewhere.  I think a busybox-based container that runs forever should be pretty straightforward.